server/src/instant/db/model/attr.clj

```diff
@@ -429,7 +421,6 @@
                                    [:not [:exists {:select :1
                                                    :from :triples
                                                    :where [:and
-                                                           :triples.ave
```

Comment:
The `triples.ave` was no help here, it just added extra work for the postgres. It would have to look at the row instead of doing an index-only scan on the triples_pkey (which has app-id, attr-id, and entity-id)